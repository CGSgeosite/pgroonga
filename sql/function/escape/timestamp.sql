SELECT pgroonga.escape(TIMESTAMP '2016-11-29 01:23:45');
SELECT pgroonga.escape(TIMESTAMP '2016-11-29 01:23:45+09');
