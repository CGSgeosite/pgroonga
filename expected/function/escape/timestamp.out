SELECT pgroonga.escape(TIMESTAMP '2016-11-29 01:23:45');
   escape   
------------
 1480382625
(1 row)

SELECT pgroonga.escape(TIMESTAMP '2016-11-29 01:23:45+09');
   escape   
------------
 1480382625
(1 row)

